.changeset/many-deers-do.md

```
@@ -0,0 +1,5 @@
+---
+"@assistant-ui/react": patch
+---
+
+feat: widen initialMessages type to ThreadMessageLike
```

Markdownlint (0.37.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)